rest/api_test.go

```diff
@@ -1050,11 +1050,8 @@ func TestLocalDocExpiry(t *testing.T) {
 	response := rt.SendAdminRequest("PUT", "/db/_local/loc1", `{"hi": "there"}`)
 	RequireStatus(t, response, 201)
 
-	cbStore, ok := base.AsCouchbaseStore(rt.Bucket())
-	require.True(t, ok)
-
 	localDocKey := db.RealSpecialDocID(db.DocTypeLocal, "loc1")
-	expiry, getMetaError := cbStore.GetExpiry(localDocKey)
+	expiry, getMetaError := rt.Bucket().DefaultDataStore().GetExpiry(localDocKey)
```

Comment:
The code in special_docs.go looks like it's using dataStore, not DefaultDataStore()?

Reply:
will follow-up on this separately